tests/lightgbm/test_lightgbm_autolog.py

```diff
@@ -387,6 +387,60 @@ def record_metrics_side_effect(self, metrics, step=None):
     assert "train-multi_logloss" in patched_metrics_data
 
 
+@pytest.mark.large
+def test_lgb_autolog_batch_metrics_logger_logs_ranking_metrics(bst_params, train_set):
```

Comment:
Can we simplify the test?

Reply:
I think we can do the same thing as `test_xgb_autolog_atsign_metrics`:

1. Enable autologging
2. Train
3. Fetch the latest test run logged in the previous step.
4. Extract the metrics from the run and confirm `@` was converted to `_at_`.

Reply:
I pushed a commit to simplify the test.